pages/about.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles/about.css">
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <nav class="top-nav">
+            <ul class="nav-item">
+            <li><a href="#"><img class="nav-img" src="/assets/facebook.png" alt="facebook"></a></li>
+            <li><a href="#"><img class="nav-img" src="/assets/twitter-.png" alt="twitter"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul></nav>
+        <nav class="nav-bar">
+            <a class="d-logo" href="../index.html"><span class="logo-span">Y-</span>DEV</a>
+            <a href="#"><img class="humberger" src="/assets/menu.svg" alt="humberger"></a>
+            <ul class="d-nav">
+                <li><a href="../index.html">Home</a></li>
+                <li><a class="nav-links" href="#">Program</a></li>
+                <li><a class="nav-links" href="#">Join</a></li>
+                <li><a class="nav-links" href="#">Sponsor</a></li>
+                <li><a class="nav-links" href="#">News</a></li>
+            </ul>
+            <button class="d-btn">cc campaign</button>
+        </nav>
+        <ul class="mobile-nav">
+            <li><a href="../index.html">Home</a></li>
+             <li><a href="#program">Program</a></li>
+             <li><a href="#join">Join</a></li>
+             <li><a href="#sponsor">Sponsor</a></li>
+             <li><a href="#news">News</a></li>
+     </ul>
+     <div class="times">&times</div>
+    </header>
+    <section class="about-hero">
+        <p class="p-1">"Hello! Dev World"</p>
+        <h1 class="h-1">YOUNG DEV'S GLOBAL SUMMIT UGANDA</h1>
+        <p class="p-2">The Young Dev Summit brings together the community of  young,expert devs.
+            this year's summit is aimed to spreading the love of tech to the young generation
+            With the help of our partners
+         </p>
+         <p class="p-3">
+            Please contact us per Email for further questions about Young Dev's Global Summit
+         </p>
```

Comment:
- [ ] Kindly make your about page font bolder as in the home page, align the text to center to match the design:
- [ ] Kindly give your text a lighter background color, right now, it cannot be readible:
![image](https://user-images.githubusercontent.com/91486791/234961509-91916aaf-1ebd-4446-a0c2-59970af56c67.png)
![image](https://user-images.githubusercontent.com/91486791/234961599-557e9c97-0c62-469d-ae27-d5780c2dcc19.png)